src/Angular.js

```diff
@@ -1903,6 +1904,27 @@ function createMap() {
   return Object.create(null);
 }
 
+function stringify(value) {
+  if (value == null) { // null || undefined
+    return '';
+  }
+  switch (typeof value) {
+    case 'string':
+      break;
+    case 'number':
+      value = '' + value;
+      break;
+    default:
+      if (hasCustomToString(value) && !isArray(value) && !isDate(value)) {
```

Comment:
They were JSONified in `$interpolate`, but `toString`ified in `ngBind`. Since we are making the two consistent, we need to break one of them.

But you convicned me: a universal format is better than forcing American English for all users.
(For some reason, I was under the impression `toString()` returned a localized string...)